src/pytest_cases/case_parametrizer_new.py

```diff
@@ -266,7 +267,11 @@ def get_all_cases(parametrization_target,  # type: Callable
         filters += (filter,)
 
     # parent package
-    caller_module_name = getattr(parametrization_target, '__module__', None)
+    if parametrization_target is None:
+        caller_module_name = getattr(parametrization_target, '__module__', None)
+    else:
+        caller_module_name = get_caller_module()
```

Comment:
This seems reverted so I guess the tests won't pass :) (if they do, it means they do not properly cover the new feature) 
Besides, `get_caller_module` would rather need to be called only when needed (maybe later in the code then ? I did not check what happens next) rather than always.